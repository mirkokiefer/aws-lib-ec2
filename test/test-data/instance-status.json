{
  "@": {
    "xmlns": "http://ec2.amazonaws.com/doc/2012-12-01/"
  },
  "requestId": "45cc6242-2a3e-44d2-885c-0c7c3cef3d11",
  "instanceStatusSet": {
    "item": {
      "instanceId": "i-74136010",
      "availabilityZone": "us-east-1a",
      "instanceState": {
        "code": "16",
        "name": "running"
      },
      "systemStatus": {
        "status": "initializing",
        "details": {
          "item": {
            "name": "reachability",
            "status": "initializing"
          }
        }
      },
      "instanceStatus": {
        "status": "initializing",
        "details": {
          "item": {
            "name": "reachability",
            "status": "initializing"
          }
        }
      }
    }
  }
}