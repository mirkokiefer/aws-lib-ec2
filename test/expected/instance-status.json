{
  "instanceId": "i-74136010",
  "instanceState": {
    "code": "16",
    "name": "running"
  },
  "systemStatus": {
    "status": "initializing",
    "details": {
      "name": "reachability",
      "status": "initializing"
    }
  },
  "instanceStatus": {
    "status": "initializing",
    "details": {
      "name": "reachability",
      "status": "initializing"
    }
  }
}